lib/filesystem/Filesystem.cpp

```diff
@@ -147,7 +147,7 @@ ISimpleResourceLoader * CResourceHandler::createInitial()
 	for (auto & path : VCMIDirs::get().dataPaths())
 	{
 		if (boost::filesystem::is_directory(path)) // some of system-provided paths may not exist
-			initialLoader->addLoader(new CFilesystemLoader("", path, 0, true), false);
+			initialLoader->addLoader(new CFilesystemLoader("", path, 1, true), false);
```

Comment:
No support nested folders? Only one level?

Reply:
Interestingly, `1` is fully sufficient here. It still works over several levels.